components/admin_console/admin_definition.jsx

```diff
@@ -1176,7 +1176,27 @@ const AdminDefinition = {
                         label_default: 'Enable Server-Side Encryption for Amazon S3:',
                         help_text: t('admin.image.amazonS3SSEDescription'),
                         help_text_markdown: true,
-                        help_text_default: 'When true, encrypt files in Amazon S3 using server-side encryption with Amazon S3-managed keys. See [documentation](!https://docs.mattermost.com/configure/configuration-settings.html#session-lengths) to learn more.',
+                        help_text_default: 'When true, encrypt files in Amazon S3 using server-side encryption with Amazon S3-managed keys. See {documentation} to learn more.',
+                        help_text_values: {
+                            documentation: (
+                                <b>
+                                    <FormattedMessage
+                                        id='admin.image.amazons3.sessionlengths'
+                                        defaultMessage='<link>documentation</link>'
+                                        values={{
+                                            link: (msg: React.ReactNode) => (
+                                            <a 
+                                                href='https://docs.mattermost.com/configure/configuration-settings.html#session-lengths' 
+                                                referrer='noreferrer' 
+                                                target='_blank'
+                                            >
+                                                {msg}
+                                            </a>),
+                                        }}
+                                    />
+                                </b>
+                            ),
+                        },
```

Comment:
Actually, since we're going to be adding the extra attributes to `<a>` repeatedly to have it open in a new tab, maybe we could define an `ExternalLink` reusable component either just in this file or in `components/external_link.tsx` for use everywhere. Then, these could just be:
```suggestion
                        help_text_default: 'When true, encrypt files in Amazon S3 using server-side encryption with Amazon S3-managed keys. See <link>documentation</link> to learn more.',
                        help_text_values: {
                            link: (msg) => (
                                <ExternalLink href='https://docs.mattermost.com/configure/configuration-settings.html#session-lengths'>{msg}</ExternalLink>
                            ),
                        },
```